### PR TITLE
Rename ExpressLRS "passphrase" to "binding phrase"

### LINKED ISSUE #3255: Rename ExpressLRS "passphrase" to "binding phrase"
### Is your feature request related to a problem? Please describe

In ExpressLRS Configurator and documentation they use the name _binding phrase_, I think it would be more consinstent if Betaflight Configurator also used the same instead of _passphrase_.

### Describe the solution you'd like

Change it at least in https://github.com/betaflight/betaflight-configurator/pull/3097/files#diff-8aa9190a05814c6894dde1e917c699ad0a2951547c963586a884eefb2d918b84R2196

A bit more ambitious is to change it also in the code.

### Describe alternatives you've considered

Keep it as is.

### Other information

Support for ExpressLRS UID was introduced in https://github.com/betaflight/betaflight-configurator/pull/3097

### PR DESCRIPTION
**Pull request by rickard-von-essen**
Rename ExpressLRS _passphrase_ to _binding phrase_ to be coherent with ExpressLRS Configurator and documentation.

~~Note: this probably puts the translations a bit off since the most likely translated _passphrase_, for example the French (fr) translation reads _"Phrase secrète"_.~~

Closes #3255

### CODE REVIEW

**Comment by rickard-von-essen** [comment]
I did not squash these two commits since I wanted to give the option of only picking 6fcfd0e13cc3e851f3acbd60eb8251ac2a9c5dbb.

**Comment by haslinghuis** [comment]
You only have to change the english label. Other languages are updated automatically using [crowdin](https://crowdin.com/).

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: locales/it/messages.json
Comment (McGiverGim) [inline locales/it/messages.json:2086]: You are breaking the translations. You are changing the "key" of the translation, that will let the translation unused.
Simply don't touch the messages files (except the en language, of course) and they will be removed automatically in Crowdin.

Comment (rickard-von-essen) [reply locales/it/messages.json]: Ok, removed all changes to locales except for `locales/en/messages.json`.

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:35]: If you add it maybe also change to camel case

Comment (rickard-von-essen) [reply src/js/tabs/receiver.js]: Done in 3fcf81d67faa972c6e91c4547b456546c5623c09

**Review by chmelevskij — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
